Godeps/Godeps.json

```diff
@@ -435,82 +462,82 @@
 		},
 		{
 			"ImportPath": "github.com/vmware/govmomi",
-			"Comment": "v0.1.0-454-g9051bd6",
+			"Comment": "v0.6.2",
```

Comment:
AFAI can see this is only used for VMWare driver, and should supposedly be left untouched when making AWS changes?